### PR TITLE
r: add head url

### PR DESCRIPTION
**Pull request by randy3k**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/r.rb [hunk 1]
```diff
@@ -4,6 +4,7 @@ class R < Formula
   url "https://cran.r-project.org/src/base/R-3/R-3.6.3.tar.gz"
   sha256 "89302990d8e8add536e12125ec591d6951022cf8475861b3690bc8bf1cefaa8f"
   revision 1
+  head "https://stat.ethz.ch/R/daily/R-devel.tar.gz"
```
Comment (SMillerDev) [inline Formula/r.rb:7]: This seems like an odd URL, HEAD should be a git repo.

Comment (bayandin) [reply Formula/r.rb:7]: I found this link to their svn:
```suggestion
  head "https://svn.r-project.org/R/trunk/"
```

**Comment by randy3k** [comment]
I have switched to an unofficial mirror: https://github.com/wch/r-source.git
The office svn repo is too slow to use.

**Comment by randy3k** [comment]
I switched back the `https://stat.ethz.ch/R/daily/R-devel.tar.gz` at the end because the svn repo is too slow and the github mirror is not complete.

**Comment by SMillerDev** [comment]
That's not what `head` is for so we can't merge it like that.

**Comment by randy3k** [comment]
Ok. It’s fair.

**Pull request closed without merging**